rep-2000.rst

```diff
@@ -892,43 +892,43 @@ Minimum language requirements:
-+-------------+----------------+----------------+----------+---------------+------------------+---------------------+
-|                                         **RMW DDS Middleware Providers**                                          |
-+-------------+----------------+----------------+----------+---------------+------------------+---------------------+
-| Cyclone DDS |                                          0.8.x (Réplique)                                           |
-+-------------+----------------+----------------+----------+---------------+------------------+---------------------+
-| Fast-DDS    |                                                2.4.x                                                |
-+-------------+----------------+----------------+----------+---------------+------------------+---------------------+
-| Connext DDS |              5.3.1              |   N/A    |     5.3.1     |                  N/A                   |
-+-------------+----------------+----------------+----------+---------------+------------------+---------------------+
-| Gurum DDS   |              2.7.x              |                         N/A                                       |
-+-------------+----------------+----------------+----------+---------------+------------------+---------------------+
++-------------+---------------------------------+------------------------------------------------------------------------------------+
+|             |        Required Support         |                        Recommended Support                                         |
++-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| Package     |  Ubuntu Jammy  |  Windows 10**  |  RHEL 8  |  Ubuntu Focal  |    macOS**    | Debian Bullseye  |   OpenEmbedded**    |
++=============+================+================+==========+================+===============+==================+=====================+
+| CMake       |     3.21.3     |     3.22.0     |  3.18.2  |     3.16.3     |    3.14.4     |      3.18.4      | 3.16.1 / 3.12.2***  |
++-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| EmPY        |      3.3.4     |                                                3.3.2                                                |
++-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| Gazebo      |    11.x.x*     |       N/A      |   N/A    |    11.0.0*     |    11.x.x     |      11.x.x*     |         N/A         |
++-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| Ignition    |    Fortress*   |       N/A      |   N/A    |    Edifice*    |   Fortress*   |     Fortress*    |         N/A         |
```

Comment:
@chapulina @scpeters Is Ignition Fortress supported on Ubuntu Focal (20.04)?

Reply:
Fixed in https://github.com/ros-infrastructure/rep/pull/342/commits/3d6a9611218a8b947070c41792e6c71f5f654cc8.